tests/rptest/tests/random_node_operations_test.py

```diff
@@ -84,11 +85,6 @@ def _setup_test_scale(self, num_to_upgrade):
         self.producer_timeout = 180
         self.consumer_timeout = 180
 
-        if num_to_upgrade > 0:
-            # we can only enable controller snapshot if all nodes are the newest version
-            cleanup_on_early_exit(self)
-            return
-
```

Comment:
nit: for the containerized setup, it seems like we're writing 50MiB across 32 partitions, but set local retention to be 3MiB (3 1MiB segments). Perhaps we should drop the partition count, or decrease the segment size even further with `log_segment_size_min`

Reply:
i will update the scale in follow up PR